### PR TITLE
Add bicep file for kusto cluster and ingestion

### PR DESCRIPTION
**Pull request by hallipr**

### CODE REVIEW

File: tools/pipeline-witness/infra/bicep/everything.bicep [hunk 1]
```diff
@@ -0,0 +1,255 @@
+param location string = 'westus2'
+param storageAccountName string = 'azsdkengsyspipelinelogs'
+param eventgridTopicName string = 'azsdkengsyspipelinelogs-53d9a67c-c733-4988-b9af-db8a0db1434c'
+param aventHubNamespaceName string = 'ADX-EG-azsdkengsys'
```
Comment (benbp) [inline tools/pipeline-witness/infra/bicep/everything.bicep:4]:
```suggestion
param eventHubNamespaceName string = 'ADX-EG-azsdkengsys'
```

File: tools/pipeline-witness/infra/bicep/everything.bicep [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+        blob: {
+          keyType: 'Account'
+          enabled: true
+        }
+      }
+      keySource: 'Microsoft.Storage'
+    }
+    accessTier: 'Hot'
+  }
+  resource blobServices 'blobServices' = {
```
Comment (benbp) [inline tools/pipeline-witness/infra/bicep/everything.bicep:75]:
```suggestion
  }

  resource blobServices 'blobServices' = {
```

File: tools/pipeline-witness/infra/bicep/everything.bicep [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+  location: location
+  properties: {
+    source: storageAccount.id
+    topicType: 'microsoft.storage.storageaccounts'
+  }
+}
+
+// Event Hub
+resource eventhubNamespace 'Microsoft.EventHub/namespaces@2022-01-01-preview' = {
+  name: aventHubNamespaceName
```
Comment (benbp) [inline tools/pipeline-witness/infra/bicep/everything.bicep:113]:
```suggestion
  name: eventHubNamespaceName
```

**Comment by benbp** [comment]
Looking good! I like the tables array declaration and iteration. Simplifies the template a ton.

File: tools/pipeline-witness/infra/bicep/everything.bicep (hunk 3 above)
Comment (hallipr) [reply tools/pipeline-witness/infra/bicep/everything.bicep:113]: thanks

**Comment by hallipr** [comment]
content's not currently covered by any build.  Overriding check enforcer

**Review by benbp — APPROVED** [review]

**Pull request merged**